examples/angular-cli/README.md

```diff
@@ -0,0 +1,162 @@
+```
+
+Then go inside the project's folder and install **TailwindCSS**, **PostCSS**, **PurgeCSS** and **Custom webpack builders for Angular** which should allow us customizing webpack configurations without ejecting the app:
+
+```sh
+cd  angular-cli
+npm i tailwindcss postcss-import postcss-loader postcss-scss @fullhuman/postcss-purgecss @angular-builders/custom-webpack -D
+```
+
+_(note: Angular-Cli already includes **autoprefixer**)_
+
+Next, create 2 webpack config files named respectively **webpack.config.dev.js** and **webpack.config.prod.js**. The main idea is to only use **PurgeCSS** when building the final prod version:
+
+_webpack.config.dev.js_
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        loader: "postcss-loader",
+        options: {
+          ident: "postcss",
+          syntax: "postcss-scss",
+          plugins: () => [
+            require("postcss-import"),
+            require("tailwindcss"),
+            require("autoprefixer")
+          ]
+        }
+      }
+    ]
+  }
+};
+```
+
+_webpack.config.prod.js_
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        loader: "postcss-loader",
+        options: {
+          ident: "postcss",
+          syntax: "postcss-scss",
+          plugins: () => [
+            require("postcss-import"),
+            require("tailwindcss"),
+            require("autoprefixer"),
+            require("@fullhuman/postcss-purgecss")({
+              // Specify the paths to all of the template files in your project
+              content: [
+                "./src/**/*.html",
+                "./src/**/*.ts"
+                // etc.
+              ],
+              // Include any special characters you're using in this regular expression
+              defaultExtractor: content =>
+                content.match(/[A-Za-z0-9-_:/]+/g) || []
+            })
+          ]
+        }
+      }
+    ]
+  }
+};
+```
+
+Then in **angular.js** file, configure the app to use them:
+
+```diff
+"architect": {
+  "build": {
+-   "builder": "@angular-devkit/build-angular:browser",
++   "builder": "@angular-builders/custom-webpack:browser",
+    "options": {
++     "customWebpackConfig": {
++       "path": "./webpack.config.dev.js"
++     },
+      "outputPath": "dist/angular-cli",
+      "index": "src/index.html",
+      "main": "src/main.ts",
+
+      (...)
+
+    "configurations": {
+      "production": {
++       "customWebpackConfig": {
++         "path": "./webpack.config.prod.js"
++       },
+        "fileReplacements": [
+          {
+            "replace": "src/environments/environment.ts",
+            "with": "src/environments/environment.prod.ts"
+          }
+```
+
+Make those changes for both **architect.build** and **architect.serve**.
+
+Next, create the Tailwind config file:
+
+```sh
+npx tailwind init
+```
+
+Finally, add this to `src/styles.css`:
+
+```scss
+@import "tailwindcss/base";
```

Comment:
Can't this be using @tailwind ?
```suggestion
@tailwind"tailwindcss/base";
```

Reply:
Did this work for you? With my current setup only `@import` worked.

Reply:
It is also the recommended way in [docs](https://tailwindcss.com/docs/installation#2-add-tailwind-to-your-css):

> If you're using postcss-import (or a tool that uses it under the hood, such as Webpacker for Rails), use our imports instead of the @tailwind directive to avoid issues when importing any of your own additional files

Reply:
Yes, I mean @tailwind base; I think the difference is in the fact I'm not using postcss-import in my project. I followed the installation guide including the [Webpack guide](https://tailwindcss.com/docs/installation/#webpack).

I have pushed my [project ](https://github.com/DeluxZ/tailwindcss-angular) for reference. Don't know what the best way to go is as I'm relatively new to TailwindCSS and PostCSS. Maybe @adamwathan can give some advice here